canonicalwebteam/image_template/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 
 def image_template(
-    url, alt, width, height, hi_def, fill=False, loading="lazy", attrs={}
+    url, alt, width, hi_def, height=None, fill=False, loading="lazy", attrs={},
```

Comment:
I see why you have to change the position of the `height` argument but IMO width and height should be always next to each other. Could `hi_def` be after `alt`?

Reply:
Done.